docs/sources/reference/api/docker_remote_api.md

```diff
@@ -46,6 +46,22 @@ You can still call an old version of the API using
 
 ### What's new
 
+`GET /containers/(id)/copy`
+
+`PUT /containers/(id)/copy`
```

Comment:
@jlhawn I know PUT is the right to use here if you want to stick to pure REST, but our current api is quite simple, we only use POST, GET and DELETE

Can you change this to use post ?